imops/measure.py

```diff
@@ -73,6 +85,8 @@ def label(
 
     if ndim > 3:
         warn("Fast label is only supported for ndim<=3, Falling back to scikit-image's implementation.")
+        if dtype is not None:
+            warn("Explicitly passed `dtype` has no effect if fallback to scikit-image's implementation has occurred.")
```

Comment:
this is bad. we should guarantee that the dtype the user requested will coincide with the one from the resulting array